spec/ransack/adapters/active_record/context_spec.rb

```diff
@@ -91,22 +91,22 @@ module ActiveRecord
               context: shared_context)
           end
 
-          describe '#join_associations', if: AR_version <= '4.0' do
-            it 'returns dependent join associations for all searches run
-                against the context' do
-              parents, children = shared_context.join_associations
-
-              expect(children.aliased_table_name).to eq "children_people"
-              expect(parents.aliased_table_name).to eq "parents_people"
-            end
-
-            it 'can be rejoined to execute a valid query' do
-              parents, children = shared_context.join_associations
-
-              expect { Person.joins(parents).joins(children).to_a }
-              .to_not raise_error
-            end
-          end
+          # describe '#join_associations', if: AR_version <= '4.0' do
```

Comment:
As we are not supporting AR <= 4 this test can be removed.

Reply:
ok